components/runners/ambient-runner/ambient_runner/platform/auth.py

```
@@ -384,6 +392,15 @@ async def populate_runtime_credentials(context: RunnerContext) -> None:
         if gitlab_creds.get("email"):
             git_user_email = gitlab_creds["email"]
 
+    # CodeRabbit credentials
+    try:
+        coderabbit_creds = await fetch_coderabbit_credentials(context)
+        if coderabbit_creds.get("apiKey"):
+            os.environ["CODERABBIT_API_KEY"] = coderabbit_creds["apiKey"]
+            logger.info("Updated CodeRabbit API key in environment")
+    except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)